src/components/Progress/sgds-progress.scss

```diff
@@ -0,0 +1,8 @@
+:host{
+    --sgds-progress-height: 1rem;
```

Comment:
docs in convention mention that `--componentname-subcomponentname-element-state-properties` 
no need sgds prefix

Reply:
resolved in [a9b34cc](https://github.com/GovTechSG/sgds-web-component/pull/110/commits/a9b34ccc04c97965cc602a23aac0a06bce3fbb86)